descriptors/ModuleDescriptor-template.json

```diff
@@ -223,7 +223,7 @@
     },
     {
       "id": "patron-notice-policy-storage",
-      "version": "0.8",
+      "version": "2.0",
```

Comment:
Why are we choosing now to go from an experimental interface version to a regular semantic version?

And why skip to 2.0, rather than 1.0?